### DISCUSSION TITLE
About ZeRO-Offload method

### DISCUSSION THREAD

**Discussion by simey1128** [deepspeedai/DeepSpeed#2531] [Q&A]
I wonder that is there any code which is used in experiment method of ZeRO-Offload paper.

thank you

**Comment by tjruwase** [comment]
Please see https://github.com/microsoft/DeepSpeed/discussions/2461

**Reply by simey1128** [reply]
Then to train GPT-2 model, I have to use deep speed v0.3.0 and GPT from NVIDIA?

**Comment by tjruwase** [comment]
The general ZeRO-Offload support, excluding DPU, is available in DeepSpeed master branch. Please see this [tutorial](https://www.deepspeed.ai/tutorials/zero-offload/).
